dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/AssemblyCheckFilterProcessorTest.java

```diff
@@ -0,0 +1,191 @@
+                                                                             1330026L, 1330028L, Orientation.REVERSE,
+                                                                             null, "1062064");
+
+        assertNull(assemblyChecker.process(deletionWithInvalidReference));
+    }
+
+    
+    @Test
+    public void validMnv() throws Exception {
+        SubSnpCoreFields mnvWithValidReference = new SubSnpCoreFields(317288161L, Orientation.FORWARD, 431853804L,
+                                                                      Orientation.FORWARD, "NT_455997.1", 1340260L,
+                                                                      1340261L, Orientation.REVERSE,
+                                                                      LocusType.SHORTER_ON_CONTIG, "22", 2532L, 2533L,
+                                                                      "TC", "TC", "A", "A/C",
+                                                                      "NC_006109.4:g.2532_2533delGAinsT", 2532L, 2533L,
+                                                                      Orientation.REVERSE,
+                                                                      "NT_455997.1:g.1340260_1340261delTCinsA",
+                                                                      1340260L, 1340261L, Orientation.FORWARD,
+                                                                      null, "1055116");
+
+        assertNotNull(assemblyChecker.process(mnvWithValidReference));
+    }
+
+    @Test
+    public void invalidMnv() throws Exception {
+        SubSnpCoreFields mnvWithInvalidReference = new SubSnpCoreFields(317288161L, Orientation.FORWARD, 431853804L,
+                                                                        Orientation.FORWARD, "NT_455997.1", 1340260L,
+                                                                        1340261L, Orientation.REVERSE,
+                                                                        LocusType.SHORTER_ON_CONTIG, "22", 2532L, 2533L,
+                                                                        "TT", "TT", "A", "A/C",
+                                                                        "NC_006109.4:g.2532_2533delGGinsT", 2532L,
+                                                                        2533L, Orientation.REVERSE,
+                                                                        "NT_455997.1:g.1340260_1340261delTTinsA",
+                                                                        1340260L, 1340261L, Orientation.FORWARD, null,
+                                                                        "1055116");
+
+        assertNull(assemblyChecker.process(mnvWithInvalidReference));
+    }
+
+    @Test
+    public void validSnpWithNoChromosomeCoordinates() throws Exception {
+        // "RefseqToGenbankMappingProcessor" replaces the Refseq contig for an Esembl one in the "contig" field of the
```

Comment:
It should be GenBank instead of Ensembl. Also applies to `invalidSnpWithNoChromosomeCoordinates`.